### PR TITLE
dev-db/percona-xtrabackup: add Michał Zając as maintainer, add missing dep on vim-core/DBD-mysql

### PR DESCRIPTION
**Pull request by hydrapolic**
Adding a co-maintainer as agreed via https://bugs.gentoo.org/265407.

Package-Manager: portage-2.3.0

### CODE REVIEW

File: dev-db/percona-xtrabackup/metadata.xml [hunk 1]
```diff
@@ -6,6 +6,11 @@
     <name>Tomáš Mózes</name>
     <description>Proxy maintainer</description>
   </maintainer>
+  <maintainer type="person">
+    <email>manwe@manwe.pl</email>
+    <name>Michał Zając</name>
+    <description>Proxy maintainer</description>
```
Comment (mgorny) [inline dev-db/percona-xtrabackup/metadata.xml]: We no longer use those descriptions, so if you could drop all of them, would be appreciated. Basically, descriptions are to be used to say how to assign bugs.

**Comment by mgorny** [comment]
The continuous tense in commit messages feels weird. Could you state them in simple tense instead? i.e. `add Michał Zając as maintainer`, `add missing dep on vim-core, #xxxxxx`. Also please try to keep those messages short to avoid wrapping on shortlog, esp. that package name is long.

**Comment by mgorny** [comment]
@gentoo/proxy-maint

**Comment by hydrapolic** [comment]
Thank you @mgorny for the suggestions, fixed the commit messages.

**Comment by kensington** [comment]
Thanks, picked as 2e6ce881b99ecfacec33c06fae61cdd0846533e6...99ca842235fc5d4ba20ba7d10f85b916fe3dffb5.

**Comment by hydrapolic** [comment]
Thank you @kensington

**Pull request closed without merging**